### PR TITLE
LPD-27774 Taken into account ctCollectionId during DDMFormInstanceSettingsUpgradeProcess v5_3_2 of the DDMFormInstance and DDMFormInstanceVersion entities

### PR DESCRIPTION
**Pull request by p-albuquerque**
Related issue: https://issues.liferay.com/browse/LPD-27774

### CODE REVIEW

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by marcelabc** [comment]
ci:test:sf

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/upgrade/v5_3_2/test/DDMFormInstanceDDMFormInstanceSettingsUpgradeProcessTest.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+			DDMFormValuesTestUtil.createUnlocalizedDDMFormFieldValue(
+				"autosaveEnabled", "true"));
+		ddmFormValues.addDDMFormFieldValue(
+			DDMFormValuesTestUtil.createUnlocalizedDDMFormFieldValue(
+				"storageType", "[\"object\"]"));
+
+		_ddmFormInstance = DDMFormInstanceTestUtil.addDDMFormInstance(
+			ddmForm, _group, ddmFormValues, TestPropsValues.getUserId());
+
+		return _ddmFormInstance;
```
Comment (marcelabc) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-test/src/testIntegration/java/com/liferay/dynamic/data/mapping/upgrade/v5_3_2/test/DDMFormInstanceDDMFormInstanceSettingsUpgradeProcessTest.java:76]: Can you return directly from `DDMFormInstanceTestUtil.addDDMFormInstance()` ?

**Review by marcelabc — CHANGES_REQUESTED** [review]
Hi 
I finished my review. Let me know if you have questions :smile:

**Comment by p-albuquerque** [comment]
Depends on: https://github.com/brianchandotcom/liferay-portal/pull/152762

**Review by marcelabc — APPROVED** [review]
LGTM 🎉

**Comment by marcelabc** [comment]
ci:test:relevant

**Comment by carolmariaabb** [comment]
ci:forward

**Pull request closed without merging**